src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/CohortController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.CommitmentsV2.Api.Types;
+using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.CommitmentsV2.Application.Commands.AddCohort;
+using SFA.DAS.CommitmentsV2.Mapping;
+
+namespace SFA.DAS.CommitmentsV2.Api.Controllers
+{
+    [Route("api/[controller]")]
+    public class CohortController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+        private readonly IMapper<CreateCohortRequest, AddCohortCommand> _addCohortMapper;
+
+        public CohortController(
+            IMediator mediator,
+            IMapper<CreateCohortRequest, AddCohortCommand> addCohortMapper)
+        {
+            _mediator = mediator;
+            _addCohortMapper = addCohortMapper;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateCohort([FromBody]CreateCohortRequest request)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            var employer = await _mediator.Send(_addCohortMapper.Map(request));
+
+            return Ok(new AddCohortResponse
+            {
+                Id = employer.Id,
+                Reference = employer.Reference,
+                DraftApprenticeshipId = employer.DraftApprenticeshipId
```

Comment:
The Draft Apprenticeship Id is no longer needed

Reply:
addressed by https://github.com/SkillsFundingAgency/das-commitments/pull/463/commits/4da1b6344a19436b3e9b3ba569f386245564a881